lib/adapters/fs.js

```diff
@@ -14,7 +15,7 @@ module.exports = objectAssign({}, FS, {
 
   writeFile: fs.writeFile,
 
-  mkdir: fs.mkdir,
+  mkdir: mkdirp,
```

Comment:
There's nothing wrong with installing another package for dev. For production, the overhead it brings should be measured.

For creating files and folders in tests we can and should use a temporary file/dir generator such as [tmp](https://www.npmjs.com/package/tmp).